src/assets/css/main.css

```diff
@@ -20,42 +20,49 @@
   font-weight: 400;
 }
 :root {
-  --main-color: #ffffff;
   --transparent-white: rgba(255, 255, 255, 0);
+  --main-color: #ffffff;
```

Comment:
Looks like all the variables are named by color except this one, should we change this to a named color as well?